src/index.js

```diff
@@ -4,6 +4,12 @@ import React from 'react';
 import Markdown from 'remarkable';
 
 var Remarkable = React.createClass({
+  propTypes: {
+    container: React.PropTypes.string,
```

Comment:
I knwo the PR getting old, but accessing `PropTypes` from the main React package is deprecated now.